snap7/util/db.py

```diff
@@ -152,7 +152,13 @@ def parse_specification(db_specification: str) -> Dict[str, Any]:
 
     for line in db_specification.split("\n"):
         if line and not line.lstrip().startswith("#"):
-            index, var_name, _type = line.lstrip().split("#")[0].split()
+            index, *notUsed, _type = line.lstrip().split("#")[0].split()
+
+            match = re.compile(r"(\s*)(\S+)(\s*)").findall(line)
+            
+            var_name = "".join([m[0] + m[1] + m[2] for m in match[1:-1]])
```

Comment:
Since you're using regexp, why not create a pattern for the entire line? Something like that:
```python
r"""
...     (?P<index>\d+) 
...     \s+
...     (?P<var_name>.*)
...     \s+
...     (?P<_type>\S+)
...     $
...     """
```

Reply:
you mean to use the regex to directly extract the 3 strings more efficently... well nice idea

Reply:
let me try

Reply:
I did it! :)
all 50 tests are working